orchagent/routeorch.cpp

```diff
@@ -387,17 +388,17 @@ bool RouteOrch::addNextHopGroup(IpAddresses ipAddresses)
     vector<sai_attribute_t> nhg_attrs;
 
     nhg_attr.id = SAI_NEXT_HOP_GROUP_ATTR_TYPE;
-    nhg_attr.value.s32 = SAI_NEXT_HOP_GROUP_ECMP;
+    nhg_attr.value.s32 = SAI_NEXT_HOP_GROUP_TYPE_ECMP;
     nhg_attrs.push_back(nhg_attr);
 
-    nhg_attr.id = SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_LIST;
+    nhg_attr.id = SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_MEMBER_LIST;
```

Comment:
I think you need to check that eventually you created "SAI_OBJECT_TYPE_NEXT_HOP_GROUP_MEMBER" objects
which should be specified into SAI_NEXT_HOP_GROUP_ATTR_NEXT_HOP_MEMBER_LIST.